src/js/components/search/visualizations/time/TimeVisualizationSection.jsx

```diff
@@ -49,6 +59,64 @@ export default class TimeVisualizationSection extends React.Component {
         }
     }
 
+    downloadLabel = () => {
+        let periodLabel;
+        if (this.props.data.visualizationPeriod === 'fiscal_year') {
+            periodLabel = 'Year';
+        }
+        else {
+            periodLabel = capitalize(this.props.data.visualizationPeriod);
+        }
+        return `Download data by ${periodLabel}`;
+    };
+
+    downloadData = () => {
+        const data = this.props.data;
+        const ret = [];
+        if (data.visualizationPeriod === 'fiscal_year') {
```

Comment:
we should use an object keyed on `data.visualsationPeriod` w/ the arrays as the value and turn these ifs elses into one line:

```javascript
const visualizationPeriodByPeriod {
   fiscal_year: ['some', 'strings']
};

data.visualizationPeriod = visualizationPeriodByPeriod[data.visualizationPeriod]
```

Reply:
done

Reply:
Since none of the following lines are simple string assignments, I am just not picturing how this would work. Let's chat.